src/PhpSpec/Matcher/ThrowMatcher.php

```diff
@@ -121,11 +121,25 @@ public function verifyPositive($callable, array $arguments, $exception = null)
         }
 
         if (!$exceptionThrown instanceof $exception) {
```

Comment:
Not in the technical sense, in that it isn't necessarily an `Exception`, but I'd guess most people would consider "exception" to mean that. What's the alternative — `$errorOrException`?!